### DISCUSSION TITLE
T&T WG Code of Conduct

### DISCUSSION THREAD

**Discussion by raghavrmadya** [filecoin-project/notary-governance#830] [T&T WG]
The goals of the T&T WG is the following:

1. Support and provide critical feedback on the tooling being built to address DataCap abuse
2. Suggest new tooling/governance changes to help clients, notaries, and SPs build trust within the Filecoin Plus program
3. Flag potential violations of the Filecoin Plus program guidelines as they relate to DC abuse
4. Create a safe space for members of the Filecoin Network and the subset of stakeholders active within the Filecoin Plus program to share their voice and concerns. 

To achieve the aforementioned goals, it is important that the T&T WG is an environment that is friendly, safe, and accepting; free from intimidation or harassment. Towards this end, certain behaviors and practices, as outlined below, will not be tolerated within the T&T WG.

1. Any reference to one's ethnicity, race, nationality, or physical appearance in a discussion post either on Slack or Github. This includes DMs and private channels

2. Calling any community member terms such as "fraud, cheat, liar" etc. **It does not matter if the accusations are supported by objective evidence**. We understand that often emotions run high during an active discussion but rather than responding on slack, it is encouraged to share your evidence in relevant forums such as the notary governance repository as an issue or discussion or bring your concern to the T& WG calls.

3. Any behavior that is deemed discriminatory" or "hate speech" will not be tolerated within the T&T WG slack or calls. The subjective terms such as "discriminatory" and "hateful" will be decided at the sole discretion of the T&T DAO.

If any member is found to be violating the aforementioned code of conduct, the T&T Lead reserves the right to remove such a post or intervene in the discussion.

**Scope**
We expect all members of the T&T WG community to abide by this Code of Conduct at all times in all discussion forums, online and in person, and in one-on-one communications pertaining to T&T affairs.

This Code of Conduct is in addition to, and does not in any way nullify or invalidate, any other terms or conditions related to the use of Filecoin services.

Reporting Violations of this Code of Conduct
If you believe someone is harassing you or has otherwise violated this Code of Conduct, please contact us at [abuse@filplus.org] to send us an abuse report. You can also DM the T&T WG Lead, currently RG on slack, with the initial report of a problem. Please include as much detail as possible. It is easiest for us to address issues when we have more context.

**Comment by TakiChain** [comment]
@raghavrmadya Can you please point out for us which words are not allowed in our community? For example, is 'abuse' a applicable word?

**Comment by raghavrmadya** [comment]
@TakiChain, we do not have a list of words that are banned. I believe most community members should practice their judgement to assess which word's violate code of conduct but if the proposal passes, disputes will be taken care by T&T DAO

**Comment by raghavrmadya** [comment]
abuse@filplus.org is not active yet. If the proposal passes, we will proceed with creating this email

**Comment by dannyob** [comment]
@raghavrmadya would it make sense, rather than creating a new CoC fresh, we could adopt a battle-hardened industry standard code , such as the [Contributor Covenant ](https://www.contributor-covenant.org/)or [PLN's CoC](https://protocol.almanac.io/docs/pln-code-of-conduct-ymBUYyonmhfvizGu6yOpXH1qkuWYce96)?
